docroot/themes/custom/govcms_uikit_starterkit/js/sitewide_search.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Sitewide search widget.
+ */
+(function ($, Drupal, window, document) {
+  'use strict';
+
+  // Example of Drupal behavior loaded.
+  Drupal.behaviors.sitewideSearch = {
+    attach: function (context, settings) {
+      if (typeof context['location'] !== 'undefined') { // Only fire on document load.
+        var $searchBlock = $('#block-exposedformsearchpage-2');
+
+        // Show the keywords input on first click of the search button.
+        $searchBlock.find('.form-submit').click(function (e) {
+          if (!$searchBlock.hasClass('is-active')) {
+            $searchBlock.addClass('is-active');
+            e.preventDefault();
+          }
+        });
+
+        $(document).click(function (e) {
```

Comment:
Hey @yusufhm I'd always be cautions about doing this in JS as this will add a click listener to all DOM elements in the page. This can compound based on the number of elements you have; even if the handler does nothing it can still cause unexpected performance degradation.

I would always recommend limiting your found elements and only applying the click handler to those elements. jQuery has a very helpful method called `filter` which will allow you return a subselection of the elements.

``` javascript
$('div').filter(function() { 
  return $(this).attr('id') === 'edit-submit-search' && $(this).attr('edit-keywords') 
}).on('click', function()  { alert('do thing!') });
```